### PR TITLE
Featured & Partner Server Tabs

### LINKED ISSUE #400: [Feature Request] Featured servers/Categories for servers
**Is your feature request related to a problem? Please describe.**
I think I am not alone when it comes to BeamMP servers when I say it is hard to find consistent and quality servers or it can be hard to find content. I feel like I have only found decent servers through word of mouth.

**Describe the solution you'd like**
A featured servers list, paid promo, or even just a recommended servers list by the staff

**Describe alternatives you've considered**
If something like this in the UI is difficult to add or just not a good option, maybe a section in the BeamMP discord for servers could be a good alternative.

**Additional context**
A lot of good servers are likely overlooked just simply because they lack the promotion and outreach. A paid promotion section could be a good motivator for the staff team to help raise funds for the development team to add more features and QOL things like this

### LINKED ISSUE #401: Partnered Server Listing
**Is your feature request related to a problem? Please describe.**
Alongside the Featured servers, a "Partnered" servers list would also be nice. 

**Describe the solution you'd like**
Similar to https://github.com/BeamMP/BeamMP/issues/400 

**Additional context**
Potentially 15 servers max per user?

### PR DESCRIPTION
**Pull request by Starystars67**
This PR adds the Featured and Partnered servers tabs into the multiplayer menu.

The locales are also included but were generated using Google translate so may be contextually incorrect.

Also removed the second German translation as it appeared to be out of date.

### CODE REVIEW

File: ui/modules/multiplayer/multiplayer.js [hunk 1]
```diff
@@ -1020,6 +1110,7 @@ async function getFeatured() {
 	return new Promise(function(resolve, reject) {
 		var feat = []
 		servers.forEach(server => {
+			console.log(server)
```
Comment (OfficialLambdax) [inline ui/modules/multiplayer/multiplayer.js]: leftover debug information?

Comment (Starystars67) [reply ui/modules/multiplayer/multiplayer.js]: Haha, yep, good spot thank you.

File: ui/modules/multiplayer/multiplayer.js [hunk 2]
```diff
@@ -525,6 +526,7 @@ function($scope, $state, $timeout) {
 	$scope.$on('onServerListReceived', async function (event, data) {
 		servers = await receiveServers(data);
 		featured = await getFeatured();
+		console.log(featured)
```
Comment (OfficialLambdax) [inline ui/modules/multiplayer/multiplayer.js]: debug info

**Review by OfficialLambdax — CHANGES_REQUESTED** [review]
Still need to test it

File: ui/modules/multiplayer/multiplayer.js (hunk 2 above)
Comment (Starystars67) [reply ui/modules/multiplayer/multiplayer.js]: Thank you, fixed too :)

**Comment by Starystars67** [comment]
> Still need to test it

Please could you test and confirm functionality for yourself too, Just as a sanity check for me at least.

**Comment by OfficialLambdax** [comment]
The featured server tab is empty and the partnered server tab is broken. Thats what happens when i click it
fghjfgghjghjkghj

**Comment by Starystars67** [comment]
> The featured server tab is empty and the partnered server tab is broken. Thats what happens when i click it fghjfgghjghjkghj

Just to confirm you cleared cache and reloaded the UI?

If so, please could you get a copy of any errors in the cef dev console for me?

**Comment by OfficialLambdax** [comment]
Yes i did.. no errors, no nothing

**Comment by Starystars67** [comment]
Odd, okay thank you.

Will load it up again my side and see if I can replicate.

**Comment by Starystars67** [comment]
Now fixed.

Featured Servers:
image

Partner Servers:
image

If someone could please retest to confirm :)

**Review by OfficialLambdax — APPROVED** [review]
Appears to be working as intended

**Pull request merged**